src/movies.js

```diff
@@ -7,9 +7,9 @@ const movies = require("./data");
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(director) {
-  const allDirectors = movies.map( function(director, index) {
-      movies.indexOf(director)=== index;
+function getAllDirectors(movies) {
+  const allDirectors = movies.map( function(director, index, movies) {
+      return movies.indexOf(director) === index;
   });
```

Comment:
With map method, you say what you want to do in each iteration, In our case, you want to return movie director. 

Couple suggestions here:
- firstly the argument you pass into the function, only pass what you will be using. 
` movies.map( function(director, index, movies)`
in our case we only want to access every movie object of that array

- secondly, you only return the value that you want to see, the element you want to see insdted of previous one. So instead full movie object I only want to see movie director.

See result below:

```
const allDirectors = movies.map( function(movie) {
      return movie.director
  });
```